start/src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointTest.java

```diff
@@ -76,7 +76,7 @@ public void testEmptyInventory() {
     int expected = 0;
     int actual = obj.getInt("total");
     assertEquals("The inventory should be empty on application start but it wasn't",
-                 expected, actual);
```

Comment:
Ideally, if you don't make any code changes to the existing code, don't re-format them. The existing code are formatted this way to look good on the website.